xml/dhcp_vendor_class.xml

```diff
@@ -203,4 +212,83 @@
     <param pos="0" name="os.family" value="Windows"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^(?i)Cisco Systems Inc\. Wireless Phone (\d{4})$">
+    <description>Cisco Wireless Phone</description>
+    <example hw.model="7925">Cisco Systems Inc. Wireless Phone 7925</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.certainty" value="0.8"/>
```

Comment:
`hw.product` here should likely be `Unified Wireless IP Phone {hw.model}`

From the NIST CPE list
```
h:cisco:unified_wireless_ip_phone_7920
h:cisco:unified_wireless_ip_phone_7921g
h:cisco:unified_wireless_ip_phone_7925g
```

Similarly, `os.product` here should likely be `Unified Wireless IP Phone {hw.model} Firmware`

```
o:cisco:unified_wireless_ip_phone_7920_firmware
```

Reply:
updated

Reply:
`update_cpes.py` now reports errors for all strings that embed `{hw.model}`:

> xml/dhcp_vendor_class.xml: ERROR: Product unified_wireless_ip_phone_{hw.model} from vendor cisco invalid for CPE h and no mapping.

Is this something something to worry about? Can it be fixed?

Reply:
It is expected with the current tooling.